src/js/Controllers/UIController.js

```diff
@@ -799,7 +799,7 @@ class UIController {
     onSliderKeepContext(msgID, appID, duration) {
         clearTimeout(this.timers[msgID])
         
-        let timeout = store.getState().ui[store.getState().activeApp].resetTimeoutValue;
+        var timeout = store.getState().resetTimeout.resetPeriod;
```

Comment:
@LiliiaShlikhtLuxoft Why does variable declaration was changed to `var` from `let` ? Looks like it's immutable value. If it's so declare it as `const`

Reply:
@ValeriiMalkov fixed